src/common/config/constants.ts

```diff
@@ -38,106 +38,108 @@ export const getProductBrandingCode = () => 'redhat';
 export const POLLING_INTERVAL = 10 * 1000;
 export const EVENTS_POLLING_INTERVAL = 10 * 1000;
 
-export const HOST_ROLES: HostRole[] = [
+export const hostRoles = (t: TFunction): HostRole[] => [
   {
-    value: 'auto-assign',
-    label: 'Auto-assign',
-    description:
-      'A role will be chosen automatically based on detected hardware and network latency.',
+    value: t('ai:auto-assign'),
```

Comment:
The `value` is a key to be sent to the Backend and should not be translated (applies to all of the items in the list)